Makefile

```diff
@@ -128,7 +128,20 @@ test-ember:
 	@echo "--> Running ember tests"
 	@cd ui && yarn run test-oss
 
-ember-ci-test:
+ember-ci-test: # Deprecated, to be removed soon.
+	@echo "ember-ci-test is deprecated in favour of test-ui-browserstack"
+	@exit 1
+
+check-vault-in-path:
+	@VAULT_BIN=$$(command -v vault) || { echo "vault command not found"; exit 1; }; \
+		[ -x "$$VAULT_BIN" ] || { echo "$$VAULT_BIN not executable"; exit 1; }; \
+		printf "Using Vault at %s:\n\$$ vault version\n%s\n" "$$VAULT_BIN" "$$(vault version)"
+
+check-browserstack-creds:
```

Comment:
Not blocking: I generally tend to try to keep the number of Make targets pretty slim. So IMO, I don't know if this warrants another Make target unless there's other targets besides `test-ui-browserstack` that might depend on this logic? Just my 2 cents :-)

Reply:
This target is purely to fail faster with a more obvious error message. (It’s just sugar, things fail appropriately without it.) This could be done elsewhere (e.g. in the test) but this produces less output so seemed better from end user perspective. The test runner produces a lot of output even if it fails immediately on a simple error like this. But maybe that’s not so bad? Happy for this check to be moved or removed if it makes maintenance difficult though.

Reply:
Just re-read that... So the concern is the extra make target, not the functionality. My instinct is to decompose concerns as I go, but the proliferation of targets could be confusing I guess too? Since this PR is approved in its current state, I'll get it merged as-is to avoid another round of reviews taking up time, but will bear this concern in mind in future. Let's sync up later on how we want to approach Makefile maintenance @jfreda

Reply:
Yeah I tend to try to keep things simple and break targets out when necessary to keep things DRY or when there's use cases to separate functionality based on how things are run in CI, for example.